.github/workflows/jekyll-docker.yml

```
@@ -12,9 +12,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v4
-    - name: Build the site in the jekyll/builder container
-      run: |
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - name: Build the site in the jekyll/builder container
+        run: |
         docker run \
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)